content/kubermatic/master/operation/etcd/etcd-launcher/_index.en.md

```diff
@@ -0,0 +1,105 @@
+
+```yaml
+kubermatic:
+  controller:
+    featureGates: "EtcdLauncher=true"
+```
+Next, you need run the kubermatic-installer to update the controller deployment:
+
+```bash
+$ ./kubermatic-installer deploy --config kubermatic.yaml --helm-values values.yaml
+```
+This should update the seed controller manager deployment and enable etcd-launcher for all users clusters.
+
+{{% notice note %}}
+Once the seed controller manager is reloaded, all users clusters will get upgraded to use etcd-launcher. In seed clusters with a large number of user clusters, this might take some time depending on the applied `-max-parallel-reconcile` value (default is 10).
+{{% /notice %}}
+
+
+#### Disabling etcd-launcher
+In this mode, to disable etcd-launcher, you need to revert the changes to the `values.yaml` file, and apply again. Additionally, you will need to disable it in each user cluster flag.
+
+To disable etcd-launcher for a specific cluster you need to set the cluster level feature flag to `false`. This can be applied _before_ enabling the feature gate on the seed controller manager to prevent the cluster from being upgraded in the first place.
+
+
+# Etcd Launcher Features
+Enabling etcd-launcher enables clusters operators to perform several operational tasks that were not possible before. With the the v2.15.0 releases, etcd-launcher provides the following capabilities:
+
+## Scaling user cluster etcd ring
+Prior to version v2.15.0, the user cluster etcd ring ran as a simple and static 3-node ring. With etcd-launcher enabled, it's now possible to resize the etcd ring to increase capacity and/or availability for user clusters.
+
+Currently, the supported minimum etcd ring size is 3 nodes. This is required to maintain etcd quorum during operations. The maximum supported size is 9 nodes, as recommended by etcd upstream.
+
+To resize the etcd ring for a user cluster, you simply need to edit the cluster:
+
+```bash
+$ kubectl edit cluster 2j2q98dkzt
+```
+
+And set the required size in the cluster spec:
+
+```yaml
+spec:
+  componentsOverride:
+    etcd:
+      clusterSize: 5
+```
+
+{{% notice warning %}}
+The resizing process is currently a disruptive process. Nodes are added/removed one by one and the etcd ring is restarted to add or remove new nodes, which could disrupt the user cluster.
+{{% /notice %}}
+
+## Automated Persistent Volume Recovery
+The etcd ring is created as a Kubernetes [StatefulSet](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/). For each etcd pod, a [PV](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) is automatically created to provide persistent storage for the pods.
+
+Unfortunately, the kubernetes StatefulSet controller doesn't automatically handle cases where a PV is unavailable. In that case, manual intervention is required to remove the related PVC and reset the StatefulSet.
```

Comment:
> where a PV is unavailable

What does this mean exactly? When do PVs become unavailable (except for bugs in the CSI plugin)? Can/do we want to give an example for a failure situation here?